### PR TITLE
Add eslint config

### PR DESCRIPTION
**Pull request by RusinovAnton**
refs #33

### CODE REVIEW

**Comment by RusinovAnton** [comment]
@yola/frontend-engineering

**Comment by eugeneford** [comment]
@RusinovAnton, looks good to me. If @yola/frontend-engineering and @beck have no objections, that its good to go.

**Comment by burnjohn** [comment]
I'd like to see it in action. Good job!

**Comment by RusinovAnton** [comment]
Actually I think we could avoid adding these dependencies at all by simply write our config from them so we will not have to install it in every project that depends on jsgreat.
```
   "eslint-config-airbnb": "^16.1.0",
 +    "eslint-plugin-import": "^2.8.0",
 +    "eslint-plugin-jsx-a11y": "^6.0.2",
 +    "eslint-plugin-react": "^7.4.0"
```
https://github.com/yola/jsgreat/pull/34/commits/5d602cb16a6f34b5a267bf2f7e8d27d76ba8f5de
As you may notice in this commit I started to overried rules that don't fit our actual code style to minimize conflict and avoid rewriting to much of code just to follow that configs' rules

**Comment by RusinovAnton** [comment]
Moreover there is more overrides for jsgreat config in sitebuilderui.

https://github.com/yola/sitebuilderui/blob/e1209c6b9a669453ccd6cf48759f8e57d7ce1409/.eslintrc#L8-L18

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1765 @@
+{
+  "name": "jsgreat",
+  "version": "0.1.9",
```
Comment (beck) [inline package-lock.json]: I wonder if there is any consequence when there is a version mismatch between package.json and package-lock.json

**Comment by beck** [comment]
> avoid adding these dependencies

The eslint module locator is woefully inadequate. Reading more about it:
https://github.com/eslint/eslint/issues/3458

The maintainers don't seem very motivated to making shareable config with plugins very friendly. Their recommendation is to use peerDepenedncies or upgrade npm and hope it flattens everything correctly (both suggestions are gross and annoying).

Airbnb gets around this with hacks:
https://github.com/airbnb/javascript/blob/16faade698a185d3a84f4c2c8ef70ec4cf712936/packages/eslint-config-airbnb-base/README.md#usage

I think the right approach for us is:
* extend a version of airbnb without plugins, `{ "extends": "airbnb/base" }`
* restore the plugins we want using this approach: https://github.com/mmazzarolo/eslint-plugin-react-app/blob/2b0da6a2dcf53803a66d10fc22cfaf043786b609/index.js

Not ideal but that saves us from having to install peerdeps with every single FE app.

Finally, I think that we should abandon jsgreat and create a new repo with a more idiomatic eslint config name, I propose `eslint-config-yola`.

**Comment by beck** [comment]
Lets close this in favor of: https://github.com/yola/eslint-config-yola/issues/1

**Pull request closed without merging**